File: Exercise1/Dog.java
package Exercise1;

public class Dog {
    private String n;
    private int a;

    public Dog(String name, int age) {
        this.n = name;
        this.a = age;
    }

    public void bark() {
        System.out.println("Woof!");
    }

    public void celebrateBirthday() {
        this.a++;
        System.out.printf("HAppy Birthday! %s is now %d years old.", this.n, this.a);
        System.out.println();
    }

    public String getInfo() {
        String res = "Name: " + this.n + ", Age: " + this.a;
        return res;
    }

    public static void main(String[] args) {
        Dog d = new Dog("Yosh", 5);
        d.bark();
        d.celebrateBirthday();
        System.out.println(d.getInfo());
    }
}

File: Exercise1/Dog.py
class Dog:
    def __init__(self, name, age):
        self.__n = name
        self.__a = age

    def bark(self):
        print("Woof! Woof!")

    def celebrate_birthday(self):
        self.__a += 1
        print(f"Happy Birthday! {self.__n} is now {self.__a} years old")

    def get_info(self):
        res = f"Dog Name: {self.__n}, Age: {self.__a}"
        return res


d = Dog("Yosh", 5)
d.bark()
d.celebrate_birthday()
print(d.get_info())